src/mono/netcore/sample/HelloWorld/Makefile

```diff
@@ -0,0 +1,23 @@
+TOP=../../../../../
+DOTNET:=$(TOP)./dotnet.sh
+DOTNET_Q_ARGS=--nologo -v:q -consoleloggerparameters:NoSummary
+
+MONO_CONFIG=Release
+MONO_ARCH=x64
+
+OS := $(shell uname -o)
+ifeq ($(OS),Darwin)
+	TARGET_OS=osx
+else
+	TARGET_OS=linux
+endif
+
+publish:
+	$(DOTNET) publish -c $(MONO_CONFIG) -r $(TARGET_OS)-$(MONO_ARCH)
+
+run: publish
+	MONO_ENV_OPTIONS="--llvm" \
```

Comment:
Please continue to set `COMPlus_DebugWriteToStdErr=1`, unless that's getting set somewhere else and I missed it.